AssemblerLib/LocalToGlobalIndexMap.h

```diff
@@ -68,6 +68,12 @@ class LocalToGlobalIndexMap
     /// Returns total number of degrees of freedom.
     std::size_t dofSize() const;
 
+    /// Returns total number of local degrees of freedom for DDC.
+    std::size_t dofSizeLocal() const
+    {
+        return _mesh_component_map.getNLocalUnknowns();
+    }
+
     /// Returns total number of global degrees of freedom for DDC.
     std::size_t dofSizeGlobal() const
```

Comment:
I see, thank you for explanation. Would you please add to the documentation of `dofSizeLocal` that it is not counting the ghost nodes; I think it will assist understanding.